crud/src/test/java/com/redhat/lightblue/rest/crud/TestHealthCheckServlet.java

```diff
@@ -0,0 +1,59 @@
+package com.redhat.lightblue.rest.crud;
+
+import java.io.File;
+import java.net.URL;
+
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.test.api.ArquillianResource;
+import org.jboss.shrinkwrap.api.Filters;
+import org.jboss.shrinkwrap.api.GenericArchive;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.EmptyAsset;
+import org.jboss.shrinkwrap.api.importer.ExplodedImporter;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.jboss.shrinkwrap.resolver.api.maven.Maven;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.gargoylesoftware.htmlunit.WebClient;
+import com.gargoylesoftware.htmlunit.html.HtmlPage;
+
+@RunWith(Arquillian.class)
+public class TestHealthCheckServlet {
```

Comment:
I'm working on setting up the arquillian tests to use a servlet container instead of just CDI (which is what the current classpath is set up to do, and why this won't work).

Once that is set up, we should use the other test that already uses Arquillian to avoid the time expense of multiple test classes each setting up a container / forking them JVM and such. We can chat more about this in scrum tomorrow.